app/controllers/api/issues_controller.rb

```diff
@@ -126,9 +126,20 @@ def archive_issue
     end
 
     def assign_issue
-      user, repo, number, @assignee = params[:user], params[:repo], params[:number], params[:assignee]
-      @issue = huboard.board(user, repo).issue(number)
-        .patch 'assignee' => @assignee
+      user, repo, number, assignees = params[:user], params[:repo], params[:number], params[:assignees]
+      issue = huboard.board(user, repo).issue(number)
+      @issue = issue.add_assignees(assignees) if issue['assignees']
+      @issue = issue.patch('assignee' => assignees.first) if !issue['assignees'] 
```

Comment:
Also, not sure how much effort has historically been put into avoiding breaking clients between a push and a refresh, but it seems like it would be relatively easy to be defensive in this case:

```ruby
user, repo, number, assignees = params[:user], params[:repo], params[:number], params[:assignees]
@assignee = params[:assignee] || @assignees.first
if issue['assignees']
  @issue = issue.add_assignees(assignees)
else
  @issue = issue.patch('assignee' => @assignee)
end
render json: @issue
```